### ISSUE TITLE
v12 support?

### ISSUE THREAD

**Issue by ruant** [thecogworks/meganav#72]
Looks like v12 added some new methods that needs to be implemented in https://github.com/callumbwhyte/meganav/blob/v9/dev/src/Our.Umbraco.Meganav/PublishedContent/PublishedElementProperty.cs ??

```
A member was added to an interface without a default implementation.
Umbraco.Cms.Core.Events.IEventAggregator.Publish2(System.Collections.Generic.IEnumerable{0})
Umbraco.Cms.Core.Events.IEventAggregator.PublishAsync2(System.Collections.Generic.IEnumerable{0},System.Threading.CancellationToken)
Umbraco.Cms.Core.Models.PublishedContent.IPublishedProperty.GetDeliveryApiValue(System.Boolean,System.String,System.String)
Umbraco.Cms.Core.Models.PublishedContent.IPublishedPropertyType.ConvertInterToDeliveryApiObject(Umbraco.Cms.Core.Models.PublishedContent.IPublishedElement,Umbraco.Cms.Core.PropertyEditors.PropertyCacheLevel,System.Object,System.Boolean,System.Boolean)
Umbraco.Cms.Core.Models.PublishedContent.IPublishedPropertyType.ConvertInterToDeliveryApiObject(Umbraco.Cms.Core.Models.PublishedContent.IPublishedElement,Umbraco.Cms.Core.PropertyEditors.PropertyCacheLevel,System.Object,System.Boolean)
Umbraco.Cms.Core.Models.PublishedContent.IPublishedPropertyType.DeliveryApiCacheLevel
Umbraco.Cms.Core.Scoping.ICoreScope.Locks
Umbraco.Cms.Core.Migrations.IMigrationPlanExecutor.ExecutePlan(Umbraco.Cms.Infrastructure.Migrations.MigrationPlan,System.String)
Umbraco.Cms.Infrastructure.Search.IUmbracoIndexingHandler.RemoveProtectedContent
Umbraco.Cms.Infrastructure.Examine.IUmbracoIndex.SupportProtectedContent
```

**Comment by afabri73** [comment]
Today I upgrade Umbraco 11.5.0 to 12.3.6 and I encourred the same error.

Any update about v12 compatibility?
Because if it no updates, I must think to uninstall it and use another package.

Can you help us?
Thanks

**Comment by afabri73** [comment]
Hi callumbwhite,
you pushed a pull request (#74) to resolve this issue.
Try it and I hope this fix the problem.

Bye
Adriano

**Comment by callumbwhyte** [comment]
Hello everybody,

Thanks for your continued use & support for Meganav.

A fix including support for Umbraco v12 and v13 has been merged – you will find v4.1.0 of Meganav is now available on NuGet.

Callum

**Comment by afabri73** [comment]
Hi Callum,
unfortunately I have problems also with version 4.1.0 (umbraco v12.3.9)

Here you can find the screenshot of the error:
https://github.com/callumbwhyte/meganav/issues/73#issuecomment-2049368735
